committeeoversightapp/models.py

```diff
@@ -101,6 +101,47 @@ def is_subcommittee(self):
         else:
             return False
 
+    @property
+    def is_permanent(self):
+        if self in CommitteeOrganization.objects.permanent_committees():
```

Comment:
This check will evaluate the permanent committees queryset and do the memebrship check in Python. I think it's more efficient to do something like `if CommitteeOrganization.objects.permanent_committees().filter(slug=self.slug).exists()`, which will just return True/False.